.github/workflows/validate.yml

```diff
@@ -129,3 +129,14 @@ jobs:
       - name: Image digest
         run: echo ${{ steps.docker_build.outputs.digest }}
 
+  formatting:
+    runs-on: ubuntu-latest
```

Comment:
I would run that in one of our containers as `ubuntu-latest` has 3 different clang versions.

Reply:
Which one?